packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -20,78 +20,35 @@ contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
         superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
     }
 
-    /// TODO: Replace struct parameters and workarounds with the appropriate
-    /// types once Kontrol supports symbolic `bytes` and `bytes[]`
-    /// Tracking issue: https://github.com/runtimeverification/kontrol/issues/272
+    /// @custom:kontrol-length-equals _withdrawalProof: 10,
+    /// @custom:kontrol-length-equals _withdrawalProof[]: 600,
+    /// @custom:kontrol-length-equals data: 1000,
```

Comment:
Ah neat, looks like these comment directives are parsed to provide bounds on the inputs? Though I'm not sure I fully understand the syntax. `_withdrawalProof: X` specifies the max array length and `_withdrawalProof[]: Y` specifies the max element length? What about `data: Z`?

If so, maybe something like this is more clear?
```
/// @custom:kontrol-array-length-equals _withdrawalProof: 10,
/// @custom:kontrol-element-length-equals _withdrawalProof: 600,
```

Reply:
Got it, that all sounds good, thanks!

Reply:
We did manage to get the natspec refactoring on time for this PR! [Here's the commit](https://github.com/ethereum-optimism/optimism/pull/9530/commits/40abc60207b466f797a52a7c074854d1caecd15f) that updates the needed comments. After internally discussing, this is the formulation to which we arrived. Let me know if you've got any comments or concerns!

Reply:
That LGTM, thank you!